src/cursor/cursor.ts

```diff
@@ -973,16 +1426,281 @@ export class Cursor<
   getLogger(): Logger {
     return this.logger;
   }
+
+  // Internal methods
+
+  /** @internal Retrieve the next document from the cursor */
+  _next(callback: Callback<Document>): void {
```

Comment:
I believe this method can go away now, `next()` above can call `nextFunction` directly.